### PR TITLE
Clickable Icon UI

### PR DESCRIPTION
**Pull request by MartinRiese**
## Product Description

Ui for clickable icons

## Technical Summary

https://dimagi-dev.atlassian.net/browse/USH-3698?atlOrigin=eyJpIjoiMTc0NDJiYTdhNDdiNGUzMDg1NjIzNzdhNjkwZGI3MjMiLCJwIjoiamlyYS1zbGFjay1pbnQifQ

## Feature Flag

CASE_LIST_CLICKABLE_ICON

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/cloudcare/templates/formplayer/case_list.html [hunk 1]
```diff
@@ -117,6 +117,8 @@ <h1  aria-label="<%- title %>" tabindex="0" class="page-title"><%- title %></h1>
     <% if (!(styles[index].widthHint === 0)) { %>
       <% if (styles[index].displayFormat === 'Image') { %>
         <td class="module-case-list-column"><img class="module-icon" src="<%- resolveUri(datum) %>"/></td>
+      <% } else if (styles[index].displayFormat === 'ClickableIcon') { %>
+        <td class="module-case-list-column"><img class="module-icon" src="<%- resolveUri(datum) %>"/></td>
```
Comment (nospame) [inline corehq/apps/cloudcare/templates/formplayer/case_list.html:121]: Sort of interesting that this template is just identical to `Image`. Is there a plan to modify/add to it, or is the clickable part being handled elsewhere?

**Review by nospame — APPROVED** [review]

File: corehq/apps/cloudcare/templates/formplayer/case_list.html (hunk 1 above)
Comment (Robert-Costello) [reply corehq/apps/cloudcare/templates/formplayer/case_list.html:121]: For the moment the idea is to get the clickable icon to display. Changes to handle the click action will be added here or elsewhere. If we handle it elsewhere we would probably need to add an additional classname or id to distinguish this element from the other icon elements if those were also present.

Comment (Robert-Costello) [reply corehq/apps/cloudcare/templates/formplayer/case_list.html]: went ahead and added `btn` class so we can grab the element in the Marionette view. Additionally, users will get the extra feedback of a visible button click.

Comment (nospame) [reply corehq/apps/cloudcare/templates/formplayer/case_list.html]: Does the `btn` class apply some default bootstrap styles? (not sure whether that would be desirable or not)

Comment (Robert-Costello) [reply corehq/apps/cloudcare/templates/formplayer/case_list.html]: It adds these styles. The only problematic one is the padding. That offsets the image slightly form the header, which I'd want to fix. Adding a gif to description. Ignore the circle that appears when clicking (and the halloween themed icons). That comes from the screen capture app I use. 

.btn {
    display: inline-block;
    margin-bottom: 0;
    font-weight: normal;
    text-align: center;
    white-space: nowrap;
    vertical-align: middle;
    touch-action: manipulation;
    cursor: pointer;
    background-image: none;
    border: 1px solid transparent;
    padding: 6px 0px;
    font-size: 12px;
    line-height: 1.42857143;
    border-radius: 4px;
    -webkit-user-select: none;
    -moz-user-select: none;
    -ms-user-select: none;
    user-select: none;
}

**Review by snopoke — APPROVED** [review]

**Pull request merged**